ui/app/templates/allocations/allocation/task/fs/index.hbs

```diff
@@ -0,0 +1,14 @@
+{{partial "allocations/allocation/task/subnav"}}
```

Comment:
Something for the future, do you have any interest in removing the use of this partial? As it looks like [partials will be deprecated](allocations/allocation/task/subnav). It seems like it could be replaced by a simple component.

Reply:
I'm quite lackadaisical when it comes to dealing with deprecations. Of course it makes sense for these to become components eventually, and I won't stand in anyone's way who wants to do that work now. I just tend to wait until I have to 😛